### ISSUE TITLE
Enemy moves "in advance" are visible by hovering over the unit sometimes (GNA #25258)

### ISSUE THREAD

**Issue by wesnoth-bugs** [wesnoth/wesnoth#1597]
**[Original submission](https://gna.org/bugs/index.php?25258)** by tobefree on 2016-11-02

By hovering over an enemy unit in a 2v2 multiplayer game, I (game host) was able to see the advance-planned units of an enemy merman hunter on the official server. My teammate was able to see them too. It was a four-moves-in-advance automatic planning which happens when you click a tile that is currently too far away to move to. This was only possible during the turn of the enemy, and stopped to be visible in the next turn.

(Reproduced on Windows 10)
Release: 1.12.6
Priority: 5 - Normal
Severity: 3 - Normal

**Comment by wesnoth-bugs** [comment]
**Modified on 2016-11-07**

**tobefree added attachment:** [wesnoth-bug-movements-2016-11-07-tobefree.jpg](http://gna.org/bugs/download.php?file_id=29325)

**Comment by ToBeFree** [comment]
![wesnoth-bug-movements-2016-11-07-tobefree](https://cloud.githubusercontent.com/assets/1029811/25829327/a61e7c10-3455-11e7-876c-de4df3fc2d99.jpg)

**Comment by CelticMinstrel** [comment]
@ToBeFree Is that a repost of the GNA attachment?

**Comment by Wedge009** [comment]
It appears to be.

**Comment by CelticMinstrel** [comment]
Yeah, I looked more closely later on and determined that it was. That's why the attachment link in the first "comment" now links to the same image instead of the GNA copy.

**Comment by jostephd** [comment]
Doesn't happen in 1.14. Could someone close this?

**Comment by ToBeFree** [comment]
Only if you verified it with an older version, so that you can actually say that it is gone.

**Comment by jostephd** [comment]
Yes, I have. I checked 1.12.6 and 1.14 from git.

**Comment by ToBeFree** [comment]
Very nice, thank you very much! 😃👍

This issue has been **solved**.

**Comment by ToBeFree** [comment]
Very nice, thank you very much! 😃👍

This issue has been **solved**.

**Comment by jostephd** [comment]
@ToBeFree Thanks. Would you mind pressing the [Close Issue] button at the bottom of this page now?

**Comment by ToBeFree** [comment]
I would, and when I asked for confirmation, I was sure that I could! But I sadly appear to be unable to close my own issue. 😕
